Python/Product/Profiling/Profiling/PythonProfilerCommandService.cs

```diff
@@ -55,5 +63,22 @@ public IPythonProfilingCommandArgs GetCommandArgsFromUserInput() {
 
             return null;
         }
+
+        private async Task<PythonProfilingPackage> GetPythonProfilingPackageAsync() {
```

Comment:
@AdamYoblick @rchiodo Can both of you give a review? I verified that within PTVS, the package loads correctly and the service returns the expected arguments, but wasn't able to test calling it from outside of PTVS.